.github/workflows/release.yml

```diff
@@ -104,8 +104,39 @@ jobs:
       upload_asset_url: ${{ needs.create_release.outputs.upload_url }}
     secrets: inherit
 
-  build_and_upload_wasinn_ubuntu:
-    name: Build and upload WASI-NN on Ubuntu 22.04
+  build_and_upload_plugin_ubuntu:
```

Comment:
The CI workflow is refactored, please refer to https://github.com/WasmEdge/WasmEdge/pull/2563 for more details.